### DISCUSSION TITLE
detect click on InPlaceAdd button

### DISCUSSION THREAD

**Discussion by elenaaralla** [serenity-is/Serenity#7231] [Help]
Hi,

I would like to detect the click on inPlaceAdd button, to pass a property to the opened dialog

Is it possible?

Thank you,
elena.

![Schermata 2024-08-20 alle 16 49 30](https://github.com/user-attachments/assets/ea0a0ab8-e4c6-410c-ace7-315df3f67243)

**Comment by VictorTomaili** [comment]
You should create custom lookup editor and override the `inplaceCreateClick` function, then set edit item with your row data like `(e as any)['editItem'] = { MyProperty: "MyValue" }`. After set the row data to editItem call the `super.inplaceCreateClick()`

**Reply by elenaaralla** [reply]
> You should create custom lookup editor and override the `inplaceCreateClick` function, then set edit item with your row data like `(e as any)['editItem'] = { MyProperty: "MyValue" }`. After set the row data to editItem call the `super.inplaceCreateClick()`

Is there any example on demo app?

Thank you,
Elena.

**Reply by VictorTomaili** [reply]
There is no example of this.

**Comment by elenaaralla** [comment]
Hi @VictorTomaili,

ok; I'll try with you suggestion.

Thank you.
